File: src/main/java/com/distribution/modules/card/entity/CardApiResponse.java
package com.distribution.modules.card.entity;

import lombok.Data;

/**
 * @author ChunLiang Hu
 * @Company
 * @Project distribution
 * @Package com.distribution.modules.card.entity
 * @Description TODO(描述)
 * @create 2018/6/8-20:45
 */
@Data
public class CardApiResponse {

    private String message;
    private CardApiResults results;
    private String status;

    public boolean isSuccess() {
        return "1".equals(this.status);
    }
}

File: src/main/kotlin/com/distribution/modules/card/entity/CardApiResponse.kt
package com.distribution.modules.card.entity

import java.io.Serializable

/**
 * @author ChunLiang Hu
 * @Company
 * @Project distribution
 * @Package com.distribution.modules.card.entity
 
 * @create 2018/6/8-20:45
 */

data class CardApiResponse(
        private val serialVersionUID: Long = 1L,
        var message: String = "",

        var results: CardApiResults = CardApiResults(),

        var status: String = "",


        val isSuccess: Boolean = false
) : Serializable
